backend/homework/1/app/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,24 @@
+package org.example;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        StudentReposittory repository = new StudentReposittory(1);
+
+        // Adding a student
+        Student newStudent = new Student(1, "Bhuvan", 20, 3);
+        repository.addStudent(newStudent);
+        logger.fatal("Added student: {}", newStudent.getName());
```

Comment:
FATAL is the highest severity logging level and generally indicates messages that indicate something has broken in an application. Instead use info.